serializability-checker-compiler-plugin/src/main/scala/org/virtuslab/ash/SerializabilityCheckerCompilerPluginComponent.scala

```diff
@@ -18,52 +20,92 @@ class SerializabilityCheckerCompilerPluginComponent(
     new StdPhase(prev) {
       private val serializabilityTraitType = typeOf[SerializabilityTrait]
 
+      private val genericsNamesWithTypes = Map(
+        ("akka.actor.typed.Behavior", Seq(ClassType.Message)),
+        ("akka.actor.typed.ActorRef", Seq(ClassType.Message)),
+        ("akka.actor.typed.RecipientRef", Seq(ClassType.Message)),
+        ("akka.persistence.typed.scaladsl.ReplyEffect", Seq(ClassType.PersistentEvent, ClassType.PersistentState)),
+        ("akka.projection.eventsourced.EventEnvelope", Seq(ClassType.PersistentEvent, ClassType.PersistentState)),
+        ("akka.persistence.typed.scaladsl.Effect", Seq(ClassType.PersistentEvent)))
+
+      private val genericMethodsWithTypes = Map(
+        ("akka.actor.typed.scaladsl.ActorContext.ask", Seq(ClassType.Message, ClassType.Message)),
+        ("akka.actor.typed.scaladsl.AskPattern.Askable.$qmark", Seq(ClassType.Message)))
+
+      private val concreteMethodsWithTypes = Map(
+        ("akka.actor.typed.RecipientRef.tell", Seq(ClassType.Message)),
```

Comment:
Pls open some issue for Akka Classic stuff as well (although I'm not sure how much we'll be able to check for there)

Reply:
#59